_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,250 @@
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, $106 has been raised which Unixstickers [just
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
+projects][related-wiki] that contains a list of current GUI projects, API
+clients, plugins that take advantage of Neovim features, and projects with
+direct support for Neovim.
+
+Some of the projects might be experimental which means there might be bugs and
+the projects might still be in progress.
+
+Do you know of another project that can be listed? Don't be afraid to add a link
+yourself!
+
+## Development News
+
+### Replacing Terminal UI
+
+TODO
+
+### Builtin Terminal Emulation
+
+@tarruda has finished [his pull request][term-pr] adding a new `:terminal`
+command which when ran launches a full terminal emulator. The terminal is using
+Neovim's native buffers and windows which means the user can enter [normal
+mode][term-normal] by pressing `<C-\><C-n>`.
+
+This is meant to replace and provide more capabilities than the already existing
+Vim command, `:shell`, which just launches a shell. More explanation on the
+decision of this feature can be found in [this comment][term-comment] made by
+@tarruda.
+
+This feature is implemented using another of @leonerd's projects called
+[libvterm][term-libvterm].
+
+### True Colors
+
+With the changes to the Terminal UI described above, this allowed @tarruda to
+make a [pull request][color-pr] to add true colors to terminals that support it.
+
+A more detailed explanation of what true colors means as well as terminals that
+support it can be found in [this Gist][color-info] that was written by @XVilka.
+
+To enable true colors, set the environment variable
+`NVIM_TUI_ENABLE_TRUE_COLOR`.
+
+### Removal of "Easy Mode"
+
+By starting Vim as `vim -y` or `evim`, it would invoke "easy mode" which as
+given by [the docs][easy-vimdoc] says it starts in insert mode which allows it
+to act like a normal editor.
+
+This lesser known and infrequently used mode has been removed by @fmoralesc with
+a [pull request][easy-pr] after some [discussion][easy-discussion]. Emulating
+that feature is still possible by using the underlying
+[`'insertmode'`](http://neovim.org/doc/user/options.html#'insertmode') option
+that easy mode leveraged.
+
+The [evim.vim runtime][easy-runtime] file contained additional settings. The
+file was also removed but the details on what it enabled can still be viewed
+in Vim's documentation (but not in Neovim's) using [`:h
+evim-keys`][easy-evim-keys].
+
+### Merging of vimdiff and -d
+
+Similar to `evim`/`vim -y` described above, [`vimdiff`][vimdiff-docs] is the
+same as `vim -d` which starts Vim in diff mode.
+
+@Pyrohh created a [pull request][vimdiff-pr] that merged these two different
```

Comment:
I don't want to sound like I'm tooting my own horn, but conceptually the vimdiff PR was only half of the picture. The other half was https://github.com/neovim/neovim/pull/2008 (vimdiff just had proportionally more documentation).